force-app/main/default/lwc/marketingCloudCopadoDataTable/marketingCloudCopadoDataTable.js

```diff
@@ -381,13 +383,15 @@ export default class MarketingCloudCopadoDataTable extends LightningElement {
         // cheking reverse direction
         let isReverse = direction === "asc" ? 1 : -1;
         // sorting data
-        parseData.sort((next, prev) => {
-            next = keyValue(next) ? keyValue(next) : ""; // handling null values
-            prev = keyValue(prev) ? keyValue(prev) : "";
-            // sorting values based on direction
-            return isReverse * ((next > prev) - (prev > next));
-        });
-        this.visibleData = parseData;
+        if (parseData) {
```

Comment:
why was this extra check needed?

Reply:
null check.  when null it throw error in sort method

Reply:
CODE COMMENT. im missing a CODE COMMENT that gives contextual information.
we talked about that earlier in our call. I don't understand why you cannot just listen to what we discuss